Modelica_LinearSystems2/Internal/frequencyVector2.mo

```diff
@@ -32,8 +32,8 @@ algorithm
   // Determine f_min2, f_max2 (auto or manual)
   if autoRange then
     (w_min,w_max) := Internal.frequencyRangeBode2(Zeros, Poles);
-    f_min2 := SI.Conversions.to_Hz(w_min);
-    f_max2 := SI.Conversions.to_Hz(w_max);
+    f_min2 := Modelica.Units.Conversions.to_Hz(w_min);

```

Comment:
Why was this second step necessary? Is this a tool issue or a problematic conversion in MSL 4?

Reply:
A tool issue, probably due to changes in development version. 
I will investigate it later, but I first needed this version.